mungegithub/mungers/submit-queue_test.go

```diff
@@ -746,6 +853,42 @@ func TestSubmitQueue(t *testing.T) {
 				test.pr.Mergeable = nil
 			}
 		})
+		path = "/bar/latest-build.txt"
+		mux.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
+			if r.Method != "GET" {
+				t.Errorf("Unexpected method: %s", r.Method)
+			}
+			w.WriteHeader(http.StatusOK)
+			w.Write([]byte(strconv.Itoa(test.lastBuildNumber)))
+		})
+		for buildNumber := range test.weakResults {
+			path = fmt.Sprintf("/bar/%v/finished.json", buildNumber)
+			// workaround go for loop semantics
+			buildNumberCopy := buildNumber
+			mux.HandleFunc(path, func(w http.ResponseWriter, r *http.Request) {
+				if r.Method != "GET" {
+					t.Errorf("Unexpected method: %s", r.Method)
+				}
+				w.WriteHeader(http.StatusOK)
+				data, err := json.Marshal(test.weakResults[buildNumberCopy])
+				if err != nil {
+					t.Errorf("Unexpected error: %v", err)
+				}
+				w.Write(data)
+			})
+		}
+		for junitFile, xml := range test.gcsJunit {
+			path = fmt.Sprintf("/bar/%v/artifacts/%v", test.lastBuildNumber, junitFile)
+			// workaround go for loop semantics
```

Comment:
Isn't xml already a copy? What is the semantic issue here? (just for me knowledge)